Services/Services/ApiRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Interfaces;
+using Interfaces.Models;
+
+namespace Services
+{
+    public class ApiRequest : IApiRequest
```

Comment:
For me that name looks confusing. Could you please rename it to the f.e. LastFmService